gateway/assets/script/bootstrap.js

```diff
@@ -36,24 +36,23 @@ app.controller("home", ['$scope', '$log', '$http', '$location', '$timeout', '$md
         }
 
         $scope.invocation.request = "";
-        $scope.isFunctionReady = "Querying";
```

Comment:
This seemed to be not used so deleted it.

Reply:
Correct